### ISSUE TITLE
Logo alt text disappears when creating a new product.

### ISSUE THREAD

**Issue by RyanCoulsonCA** [ramp4-pcar4/storylines-editor#438]
## Description

When creating a new product, the logo alt text is not being saved correctly. It disappears when you proceed to the main editor and click `Edit Product Metadata`.

I'm only seeing this on product creation. If you enter the alt text at any point after this, it works as expected.

## Environment

- Browser (if applicable): Chrome

## Screenshots

![noalttext](https://github.com/user-attachments/assets/a6828952-62a8-439d-ac76-06e356410280)

## Steps to reproduce

1. Create a new product.
2. Upload a logo and set the alt text.
3. Click `Next` to proceed to the main editor page.
4. Click `Edit Product Metadata`
5. Notice that the alt text is gone.

## Expected behavior

The alt text should be saved correctly.

## Actual behavior

It's not!
